spec/runners/unified/crud_operations.rb

```diff
@@ -146,7 +146,7 @@ def insert_one(op)
         if session = args.use('session')
           opts[:session] = entities.get(:session, session)
         end
-        collection.insert_one(args.use!('document'), **opts)
+        collection.insert_one(args.use('document'), **opts)
```

Comment:
What happened here?

Reply:
One of the tests added was to test that if you supply a `nil` document to insert_one, it fails. 

We do, in fact, fail in this case, and we raise an ArgumentError. This is why I added the ArgumentError above the errors we check for.

Reply:
This can be handled in one of two ways: eitheir `use` can be modified to check for key presence via `key?` and return the value of that key even if it's nil, or the `ArgumentError` needs to be reraised as a unified spec runner exception (like the other exceptions appearing in this PR) and that specific exception can be rescued here. The second option retains an early check on values being `use`d being not nil everywhere else in the spec runners which I think is good, thus would be my preference.